tests/app/lanes.go

```diff
@@ -85,9 +85,17 @@ func CreateLanes(app *TestApp) (*mevlane.MEVLane, *base.BaseLane, *base.BaseLane
 		freeMatchHandler,
 	)
 
-	defaultLane := defaultlane.NewDefaultLane(
+	// Create a default mempool for the base lane
+	defaultMempool := base.NewMempoolWithDefaultOrdering(
```

Comment:
Updated here so e2e + integration use the "default" mempool as opposed to the "prioritynonce" mempool